presto-main/src/main/java/com/facebook/presto/operator/scalar/MathFunctions.java

```diff
@@ -526,6 +527,69 @@ private static SqlScalarFunction decimalRoundNFunction()
                 .build();
     }
 
+    private static SqlScalarFunction decimalTruncateFunction()
```

Comment:
(1) don't insert methods for a function in between methods that compose a different function (roundN). 
(2) either use xxTruncateFunction or xxTruncateNFunction, not both if there is only one function implementation